packages/ui/src/components/molecules/UpdateBar.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { Typography, Flex, Image, Button } from '../atoms';
+
+export type updateState = 'normal' | 'progress' | 'success' | 'error';
```

Comment:
Type should be in `PascalCase`, check [CODING GUIDELINES](https://github.com/Cypherock/cypherock-cysync/blob/develop/docs/CODING_GUIDELINES.md#naming-convention)